### ISSUE TITLE
curl -L https://install.meilisearch.com | sh

### ISSUE THREAD

**Issue by hbsarobin** [meilisearch/meilisearch#2151]
**Describe the bug**
Hello

When I want to install meilisearch on an amd64 architecture.
I get the following error 

```
Downloading Meilisearch binary v0.25.2 for linux, architecture amd64...
curl: option --fail-with-body: is unknown
curl: try 'curl --help' or 'curl --manual' for more information
```

**To Reproduce**
1. Install amd64
2. curl -L https://install.meilisearch.com | sh
3. See error

**Comment by curquiza** [comment]
Sorry for this! On it

**Comment by danielemontecchi** [comment]
Same error....Ubuntu 20.04 on AWS :(

**Comment by Kerollmops** [comment]
Hey @danielemontecchi, @Ramaniks, @hbsarobin,

We just released a fix for this script, would it be possible for you to test it on your machine? It should be available now.

The issue was that we wanted to make the installation script but we used a flag (i.e. `--fail-with-body`) that was introduced "recently" on curl, therefore, old versions of curl were discarding it. We changed that by using an older one (i.e. `--fail`).

Thank you very much!
